integrations/freshchat/src/definitions/freshchat-events.ts

```diff
@@ -0,0 +1,129 @@
+// Enum for actor types
+enum ActorType {
```

Comment:
**chore:** we don't use enums internally, we prefer union of literals when possible